experiment/benchmark.py

```diff
@@ -73,16 +73,18 @@ def from_yaml(cls, benchmark_path: str) -> List:
     with open(benchmark_path, 'r') as benchmark_file:
       data = yaml.safe_load(benchmark_file)
 
-    benchmark_name = os.path.splitext(os.path.basename(benchmark_path))[0]
+    project_name = os.path.splitext(os.path.basename(benchmark_path))[0]
 
     use_context = data.get('use_context', False)
     use_project_examples = data.get('use_project_examples', True)
     cppify_headers = data.get('cppify_headers', False)
     commit = data.get('commit')
     functions = data.get('functions', [])
     for function in functions:
+      max_len = os.pathconf('/', 'PC_NAME_MAX') - len('output-')
```

Comment:
Can you add a comment explaining that function signatures can be very long and exceed filesystem limits?

Reply:
added, thanks!